### PR TITLE
New Files

### PR DESCRIPTION
**Pull request by awaismemon01**

### CODE REVIEW

File: src/AbstractDemo.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+abstract class Figure{
```
Comment (ossdhaval) [inline src/AbstractDemo.java]: Please separate out these classes

File: src/InheritDemo.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
+class Box{
```
Comment (ossdhaval) [inline src/InheritDemo.java]: Please separate out these classes.

**Review by ossdhaval — CHANGES_REQUESTED** [review]

File: src/AbstractDemo.java (hunk 1 above)
Comment (awaismemon01) [reply src/AbstractDemo.java]: Done

File: src/InheritDemo.java (hunk 2 above)
Comment (awaismemon01) [reply src/InheritDemo.java]: Done

**Pull request merged**